packages/sync/src/Functions.php

```diff
@@ -1,43 +1,65 @@
 <?php
+/**
+ * Utility functions to generate data synced to wpcom
+ *
+ * @package automattic/jetpack-sync
+ */
 
 namespace Automattic\Jetpack\Sync;
 
 use Automattic\Jetpack\Constants;
-/*
+
+/**
  * Utility functions to generate data synced to wpcom
  */
-
 class Functions {
 	const HTTPS_CHECK_OPTION_PREFIX = 'jetpack_sync_https_history_';
```

Comment:
Would be great if these consts could also get documented.

Reply:
I'm open to suggestions on the most accurate wording.

Reply:
Maybe something like `Prefix of the option name, used for the scheme history.` for `HTTPS_CHECK_OPTION_PREFIX` and `Maximum number of entries in the scheme history.` for the `HTTPS_CHECK_HISTORY` var.